### ISSUE TITLE
Invalid hub connection should be logged

### ISSUE THREAD

**Issue by jayunit100** [blackducksoftware/perceptor#92]
*Problem:*

My hub was down, but I saw this in the logs, kinda implying healthy functioning:
```
time="2018-03-10T15:06:11Z" level=info msg="looking for next image to search for in hub"
time="2018-03-10T15:06:11Z" level=info msg="hub check queue empty"
time="2018-03-10T15:06:31Z" level=info msg="checking for stalled scans"
time="2018-03-10T15:06:31Z" level=info msg="checking hub for completed scans"
time="2018-03-10T15:06:31Z" level=info msg="looking for next image to search for in hub"
time="2018-03-10T15:06:31Z" level=info msg="hub check queue empty"
time="2018-03-10T15:06:51Z" level=info msg="checking hub for completed scans"
time="2018-03-10T15:06:51Z" level=info msg="looking for next image to search for in hub"
time="2018-03-10T15:06:51Z" level=info msg="hub check queue empty"
```

*Solution:*

Lets return response codes + HUB IP information when we check the queue if possible in the logs, just to make sure this information is cross-validatable - i have a suspicion it might be hiding the fact that the hub *is down*.

**Comment by mattfenwick** [comment]
@jayunit100 what did you see in the metrics?  Keep in mind that the metrics should be our first line of inquiry.  

I suspect that the metrics would show this very obviously, and much better and clearer than any logs.

**Comment by mattfenwick** [comment]
Fixed in #107 -- logs will be less noisy now.
